playbooks/tasks/portal-gzip-logs.yml

```diff
@@ -0,0 +1,24 @@
+# GZIP webportal logs
+
+# Stat all the portal_log_files.
+- name: Stat Files
+  ansible.builtin.stat:
+    path: "{{ file }}"
+  register: loop_results
+  loop: "{{ portal_log_files }}"
+  loop_control:
+    loop_var: file
+  when: portal_log_files is defined
```

Comment:
I would leave out this `when`. If `portal_log_files` are not defined user gets a message of the missing config which should be mandatory for this playbook.